### ISSUE TITLE
Please document differences between builds

### ISSUE THREAD

**Issue by TheIndifferent** [JetBrains/JetBrainsRuntime#68]
Could you please add some information about different JBR builds?

There seem to be 5 builds available (for windows for example):
https://confluence.jetbrains.com/pages/viewpage.action?pageId=221478946

SDK and fastdebug seem somewhat self-explanatory, but vanilla is not, is that vanilla openJDK runtime (without JetBrains patches) or vanilla JetBrains runtime with all the good stuff in it?

Also, while I was able to find information what id DCEVM, it would be nice to see some ELI5 if I want it as a user or not 😃

**Comment by mkartashev** [comment]
> Is vanilla and JBR the same thing with only difference name?

On the [Releases](https://github.com/JetBrains/JetBrainsRuntime/releases) page, there is no JBR flavour alone, it's always accompanied by a qualifier.

> Does "Better FPS performance; Improved font rendering, keyboard input; Robust desktop experience" is part of JCEF or JBR?

JCEF is [The Java Chromium Embedded Framework](https://github.com/chromiumembedded/java-cef), while JBR package is a Java Runtime Environment (JRE) with a host of additions compared to JRE from OpenJDK. 

Please, describe your intended usage, and I might be able to help choosing the right flavour.
